docs/usage/updating-rebasing.md

```diff
@@ -1,53 +1,59 @@
 It is up to you to either finish the job and merge the PR, or rename it and close it so that Renovate can take back over again.
 
-## Rebasing Conflicted PRs
+## Refreshing conflicted PRs
 
-If new commits to the base branch - such as merging another Renovate PR - result in an open Renovate PR having merge conflicts, then Renovate will recreate ("rebase") any conflicted PRs.
+If new commits to the base branch - such as merging another Renovate PR - result in an open Renovate PR having merge conflicts, then Renovate will refresh any conflicted PRs.
 This applies both to commits to dependency files such as `package.json` as well as lock files such as `yarn.lock`.
 You should not ever need to resolve such conflicts manually.
-You can disable this functionality by configuring `"rebaseWhen": "never"` (not recommended);
+You can disable this functionality by configuring `"refreshWhen": "never"` (not recommended).
 
-## Rebasing Out-of-date Branches
+## Refreshing out-of-date branches
 
-There are two cases where Renovate will rebase its branches off the base branch every time they are out of date:
+There are two cases where Renovate will refresh its branches off the base branch every time they are out of date:
 
-1.  If you manually configure `"rebaseWhen": "behind-base-branch"`
-2.  If you have enabled "Require branches to be up to date before merging" on GitHub protected branches settings, and `rebaseWhen` has default value `"auto"`
+1. If you manually configure `"refreshWhen": "behind-base-branch"`
+1. If you have enabled "Require branches to be up to date before merging" on GitHub protected branches settings, and `refreshWhen` has default value `"auto"`
 
-In that case Renovate PRs will be continuously rebased off the repository's base branch whenever necessary, even if the PRs are not conflicted.
+In that case Renovate PRs will be continuously refreshed off the repository's base branch whenever necessary, even if the PRs are not conflicted.
 
-## Newer Dependency Versions
+## Newer dependency versions
 
-If an existing PR is open to upgrade dependency "foo" to v1.1.0 and then v1.1.1 is released, then Renovate will regenerate the branch again.
+If you have a existing PR to upgrade dependency "foo" to 1.1.0, and later 1.1.1 is released, then Renovate will regenerate the branch.
 This way:
 
-- Each Renovate branch will always have 1 and only 1 commit
+- Each Renovate branch always has 1 and only 1 commit
 - The newest version will be based off the latest base branch commit at the time
 
-## Manual rebasing
+## Manual refresh
 
-It is possible to manually request that Renovate rebase a PR by ticking a rebase/retry checkbox on GitHub or GitLab, or by adding the label "rebase" to a Renovate PR.
-This label name is also configurable via the `rebaseLabel` config option too.
+You can request that Renovate refresh a PR by ticking the refresh checkbox on GitHub or Gitlab.
+Or you can add a "refresh" label to the PR.
```

Comment:
This is a problem because people may be used to using the `rebase` label. Maybe we need to support both?

Reply:
I'd like to just migrate everybody to the new label and make a clean break with the old term.
But this is more work to program, more error prone, and causes noise for existing users.

So, I think we have no other option but to support both labels.
We can however only mention the newer labels, so that new users are "nudged" towards using the newer labels.